src/main/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceService.java

```diff
@@ -117,11 +113,11 @@ public ObjectResponse create(String accountUrn, ObjectCreate createObject) {
         }
 
         // findByExample doesn't deal with dates, so we have to do it ourselves
-        Date modifiedAfterDate = new Date(0);
+        Long modifiedAfterDate = null;
         boolean modifiedAfterDateSpecified = false;
```

Comment:
We can't be doing things like this.  There is no need to create and use a separate variable to represent the presence of something.  Initialize the `modifiedAfterDate` field to `null` as it is here and later check it for `!= null`.  One less variable.